lib/evmone/baseline.cpp

```diff
@@ -79,17 +79,7 @@ CodeAnalysis analyze_eof1(bytes_view container)
     const auto executable_code =
         container.substr(code_sections_offset, code_sections_end - code_sections_offset);
 
-    // Code section offsets relative to the beginning of the first code section.
-    // Execution starts at position 0 (first code section).
-    // The implementation of CALLF uses these offsets.
-    CodeAnalysis::CodeOffsets relative_offsets;
-    relative_offsets.reserve(header.code_offsets.size());
-    for (const auto offset : header.code_offsets)
-        relative_offsets.push_back(offset - code_sections_offset);
-
-    // FIXME: Better way of getting EOF version.
-    const auto eof_version = container[2];
-    return CodeAnalysis{executable_code, {}, eof_version, relative_offsets};
+    return CodeAnalysis{executable_code, header};
```

Comment:
```suggestion
    return CodeAnalysis{executable_code, std::move(header)};
```
but `header` cannot be `const.

Reply:
OK. I though it's not needed here because it wasn't used for `relative_offsets` field before which was supposed to be moved too.